src/bitcoin.cr

```diff
@@ -451,7 +446,7 @@ module Secp256k1::Bitcoin
       pk_checksum = hashed_twice[0, 8]
       valid = valid && wif_checksum === pk_checksum
     end
-    return valid
+    valid
```

Comment:
i personally prefer having return statements to improve readability but I'm not very opinionated about it, so LGTM

Reply:
I actually concur, but the crystal/ruby community decided to use return only for early break outs